Neos.Flow/Classes/Configuration/ConfigurationManager.php

```diff
@@ -423,6 +423,7 @@ public function warmup()
      */
     protected function loadConfiguration($configurationType, array $packages)
     {
+        $this->configurations[$configurationType] = [];
```

Comment:
Is it impossible that `$this->configurations[$configurationType]` is already set at that point?